pages/casestudies/index.js

```diff
@@ -41,7 +42,58 @@ export default function Casestudies() {
           <CaseStudyCard studies={CaseStudiesList}
           />
         </div>
+
+        <div className="grid lg:grid-cols-9 lg:gap-8 lg:text-center mt-8">
+          <div className="col-start-3 col-span-5">
+            <Heading level="h1" typeStyle="heading-lg">
+              Adopters
+            </Heading>
+            <Paragraph typeStyle="body-md" className="my-4 max-w-4xl" >
+              Check out how different companies use AsyncAPI and what problems they solve.
+            </Paragraph>
+            <Paragraph typeStyle="body-md" className="my-4 max-w-4xl">
+              Feel free to submit a pull request with information about how your company uses AsyncAPI. We know that
```

Comment:
`submit a pull request` can you make it a link that points to `config/adopters/adoptors.yml` file, so people immediately know where to place the info